Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -5471,6 +5485,61 @@ def put_file_get_successful_action_results(client, res):
     )
 
 
+# -------------- Library ---------------
+def upload_library_file(client, args):
+    entry_ids = argToList(args['entry_id'])
+    description = args['description']
+    HasParameters = args.get("has_parameters", "false")
+    OverrideIfExists = args.get("override_if_exists", "true")
+    outputs = []
+    for entry_id in entry_ids:
+        info_file = demisto.getFilePath(entry_id)
+        file_path = info_file['path']
+        file_name = info_file['name']
+
+        f = open(file_path, 'rb')
+        file_name = urllib.parse.quote(file_name)
+        files = {
+            "file": (file_name, f)
+        }
```

Comment:
Please use `with open(file_path, 'rb') as f`.
Otherwise, the file won't be closed and will remain open.

Reply:
Changes have been made